packages/core/src/types.ts

```diff
@@ -498,6 +498,7 @@ export interface Monograph extends BaseItem<"monograph"> {
   datePublished: number;
   selfDestruct: boolean;
   password?: Cipher<"base64">;
+  slug?: string;
```

Comment:
Instead of having `slug` here, we should instead have `publishUrl` that contains the full monograph url including the domain etc. The `publishUrl` will be passed down by the server, of course.